### ISSUE TITLE
Inputs should have labels !

### ISSUE THREAD

**Issue by y-zaky** [FAC-11/week2-atoms#40]
Accessibility audit [Severe] Controls and media elements should have labels (2) 
refers to :    https://github.com/FAC-11/week2-atoms/blob/master/index.html#L20-L23

Just add some labels! 

some guidelines are here: https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_01
